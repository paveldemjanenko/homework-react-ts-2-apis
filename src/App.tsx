import React from 'react';
import { UserForm } from './components/UserForm';
import Notifications from 'react-notify-toast';
import './App.css';

function App() {
  return (
    <div className="App">
      <UserForm />
      <Notifications />
    </div>
  );
}

export default App;
