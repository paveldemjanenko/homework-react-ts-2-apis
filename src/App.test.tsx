import { render } from '@testing-library/react';
import App from './App';

test('renders user form', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
