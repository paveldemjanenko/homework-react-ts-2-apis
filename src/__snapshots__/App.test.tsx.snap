// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders user form 1`] = `
<div>
  <div
    class="App"
  >
    <form
      class="user-form"
    >
      <label>
        Name
      </label>
      <input
        name="name"
        readonly=""
        type="text"
        value=""
      />
      <label>
        Last Name
      </label>
      <input
        name="lastName"
        readonly=""
        type="text"
        value=""
      />
      <label>
        E-mail
      </label>
      <input
        name="email"
        readonly=""
        type="text"
        value=""
      />
      <label>
        Car
      </label>
      <select
        data-testid="select"
      >
        <option
          value=""
        >
          Please select a car
        </option>
        <option
          value="Golf"
        >
          Golf
        </option>
        <option
          value="Arteon"
        >
          Arteon
        </option>
        <option
          value="Tiguan"
        >
          Tiguan
        </option>
      </select>
      <label>
        Purchase Date
      </label>
      <div
        class="react-datepicker-wrapper"
      >
        <div
          class="react-datepicker__input-container"
        >
          <input
            class=""
            placeholder="Click to select a date"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        class="user-form-buttons"
      >
        <input
          type="button"
          value="Prefill User Info"
        />
        <input
          type="button"
          value="Submit"
        />
      </div>
    </form>
    <div
      id="notification-wrapper"
    />
  </div>
</div>
`;
